src/System.IO.Compression/ref/System.IO.Compression.cs

```diff
@@ -42,7 +42,11 @@ public partial class DeflateStream : System.IO.Stream
         public override IAsyncResult BeginWrite(byte[] array, int offset, int count, AsyncCallback asyncCallback, object asyncState) { throw null; }
         public override void EndWrite(IAsyncResult asyncResult) { }
         public override void Write(byte[] array, int offset, int count) { }
+        public override void Write(System.ReadOnlySpan<byte> buffer) { }
         public override System.Threading.Tasks.Task WriteAsync(byte[] array, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken = default) { throw null; }
+        public override void CopyTo(Stream destination, int bufferSize) { }
+        public override System.Threading.Tasks.Task CopyToAsync(Stream destination, int bufferSize, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
The only thing you should end up needing to add here is the one-line for the CopyTo override.  I expect these other additions are the cause of the conflict.